src/Bindings/ManualBindings_Network.cpp

```diff
@@ -878,11 +878,11 @@ static int tolua_cUDPEndpoint_Send(lua_State * L)
 
 	// Get the data to send:
 	AString Data, RemotePeer;
-	int RemotePort = 0;
+	UInt16 RemotePort = 0;
 	S.GetStackValues(2, Data, RemotePeer, RemotePort);
 
 	// Check the port:
-	if ((RemotePort < 0) || (RemotePort > USHRT_MAX))
+	if ((RemotePort < 0) || (RemotePort > std::numeric_limits<UInt16>::max()))
```

Comment:
GetStackValues will return false if the input value is not a number or falls outside the range of the type you have requested conversion to.